chapter0/chapter0.md

```diff
@@ -61,7 +61,7 @@ This jargon overload is due to the confluence of terms from Control Theory (emer
 
 It is also important to recognize that Reinforcement Learning is considered to be a branch of Machine Learning. While there is no crisp definition for *Machine Learning* (ML), ML generally refers to the broad set of techniques to infer mathematical models/functions by acquiring ("learning") knowledge of patterns and properties in the presented data. In this regard, Reinforcement Learning does fit this definition. However, unlike the other branches of ML (Supervised Learning and Unsupervised Learning), Reinforcement Learning is a lot more ambitious - it not only learns the patterns and properties of the presented data (internally building a model of the data), it also learns about the appropriate behaviors to be exercised (appropriate decisions to be made) so as to drive towards the optimization objective. It is sometimes said that Supervised Learning and Unsupervised learning are about "minimization" (i.e., they minimize the fitting error of a model to the presented data), while Reinforcement Learning is about "maximization" (i.e., RL also identifies the suitable decisions to be made to maximize a well-defined objective). Figure \ref{fig:ml_branches} depicts the in-vogue classification of Machine Learning.
 
-![Branches of Machine Learning \label{fig:ml_branches}](./chapter0/BranchesofML.jpg "Branches of Machine Learning")
+![Branches of Machine Learning \label{fig:ml_branches}](./chapter0/BranchesOfML.jpg "Branches of Machine Learning")
```

Comment:
Fixed a typo here. It worked fine on your computer because macOS doesn't care about cases in filenames, but Linux is case sensitive so it caused an error for me.